### PR TITLE
chore(commands): add documentation for commands

### PR DESCRIPTION
**Pull request by PatrickDohn**
Created markdown file for documentation of HrBot commands.

### CODE REVIEW

File: docs/hrbot-commands.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# HrBot Commands
```
Comment (davidholyko) [inline docs/hrbot-commands.md:1]: was this written by hand? Maybe later we will write some code that can generate this file

Comment (davidholyko) [inline docs/hrbot-commands.md:1]: i believe this project is `hrbot` all lowercase

Comment (PatrickDohn) [reply docs/hrbot-commands.md]: Yes, i typed it all out

Comment (PatrickDohn) [reply docs/hrbot-commands.md]: Fixed!

File: docs/hrbot-commands.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# hrbot Commands
+
+| Commands:                               |                      Description                      |
```
Comment (davidholyko) [inline docs/hrbot-commands.md:3]: rm semi colon

File: docs/hrbot-commands.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+| :-------------------------------------- | :---------------------------------------------------: |
+| how many sick days did I take this year |   Tells the user how many sick days they have taken   |
```
Comment (davidholyko) [inline docs/hrbot-commands.md:5]: its kind of strange this column is left aligned and right column is center aligned. can they both be left aligned?

Comment (PatrickDohn) [reply docs/hrbot-commands.md]: Fixed!

File: docs/hrbot-commands.md (hunk 2 above)
Comment (PatrickDohn) [reply docs/hrbot-commands.md]: Fixed!

**Review by davidholyko — APPROVED** [review]

**Pull request merged**